packages/core/src/routes/logto-config.openapi.json

```diff
@@ -188,6 +188,48 @@
           }
         }
       }
+    },
+    "/api/configs/jwt-customizer/test": {
+      "post": {
+        "summary": "Test JWT customizer",
+        "description": "Test the JWT customizer script with the given context and token payload.",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "properties": {
+                  "script": {
+                    "description": "The script to test the JWT customizer."
+                  },
+                  "envVars": {
```

Comment:
it will be better to use the full name "environmentVariables"

Reply:
Add an issue to do it later before release, the renaming includes schemas, console, core and cloud packages which takes at least two dependent changes, see LOG-8449.

Reply:
ok